### DISCUSSION TITLE
Tesla Browser time zone

### DISCUSSION THREAD

**Discussion by freshfieldx** [alandtse/tesla#410] [General]
This obviously isn't directly related to the excellent integration directly, but has anyone found a way for HA to use the correct time zone when viewed in the car? I'm in the UK and my HA graphs are 8 hours behind when viewed in my M3 web browser.

**Comment by freshfieldx** [comment]
Asking here as we are all using  Tesla!

**Comment by freshfieldx** [comment]
Must be just me!

**Comment by cvwillegen** [comment]
Not just you, it's me as well...
